datacube/utils/geometry/_base.py

```diff
@@ -707,15 +699,14 @@ def box(left, bottom, right, top, crs):
     return polygon(points, crs=crs)
 
 
-def polygon_from_transform(width, height, transform, crs):
+def polygon_from_transform(width: float, height: float, transform: Affine, crs: MaybeCRS) -> Geometry:
     """
     Create a 2D Polygon from an affine transform
 
     :param float width:
     :param float height:
     :param Affine transform:
     :param crs: CRS
```

Comment:
I would think these are redundant now too?

Reply:
yeah, I only searched to `:(r)type` annotations, it's probably ok, I'll just merge